pirk-core/src/main/java/org/apache/pirk/test/distributed/DistributedTestCLI.java

```diff
@@ -18,12 +18,7 @@
  */
 package org.apache.pirk.test.distributed;
 
-import org.apache.commons.cli.CommandLine;
-import org.apache.commons.cli.CommandLineParser;
-import org.apache.commons.cli.GnuParser;
-import org.apache.commons.cli.HelpFormatter;
-import org.apache.commons.cli.Option;
-import org.apache.commons.cli.Options;
+import org.apache.commons.cli.*;
```

Comment:
We need to take out the * imports - prob just inadvertently added in via your IDE - this is throughout the PR